docs/integrations/delta.md

```diff
@@ -15,11 +15,26 @@ has_children: false
 lakeFS is format-agnostic, so you can save data in Delta format within a lakeFS repository to reap the benefits of both technologies. Specifically:
 
 1. ACID operations can now span across many Delta tables.
-1. [CI/CD hooks](../use_cases/cicd_for_data.md#using-hooks-as-data-quality-gates) can validate Delta table contents, schema, or even referential integrity.
-1. lakeFS supports zero-copy branching for quick experimentation with full isolation.
+2. [CI/CD hooks](../use_cases/cicd_for_data.md#using-hooks-as-data-quality-gates) can validate Delta table contents, schema, or even referential integrity.
+3. lakeFS supports zero-copy branching for quick experimentation with full isolation.
 
 {% include toc.html %}
 
+## Viewing Delta table changes in lakeFS
+
+Using lakeFS, you can compare different versions of Delta tables, view the table operations done since the tables diverged and their details.
+
+
+For example, comparing branches `dev` and `main`, we can see that _movies_ table has changed on `dev` since the branches diverged. 
+Expanding the delete operation, we learn that all movies with rating < 4 were deleted from the table on the `dev` branch.
+
+
+![movies_table_changed.png](../assets/img/delta-diff-table-icon.png)
+![movies_table_operations.png](../assets/img/delta-diff-operations.png)
+
+
+**Note:** 
+The diff is available as long as the table history in Delta is retained ([30 days by default](https://docs.databricks.com/delta/history.html#configure-data-retention-for-time-travel)).
```

Comment:
I'd put this note before the "For example" in this section, since it's an important point that is currently buried below the example that ppl might skim over.